tests/test_split_cwd_inheritance.py

```
@@ -0,0 +1,213 @@
+
+def main() -> int:
+    tag = os.environ.get("CMUX_TAG", "")
+
+    socket_path = None
+    if tag:
+        socket_path = f"/tmp/cmux-debug-{tag}.sock"
+    client = cmux(socket_path=socket_path)
+    client.connect()
+
+    # Use resolved paths to avoid /tmp -> /private/tmp symlink mismatch on macOS
+    test_dir_a = str(Path("/tmp/cmux_split_cwd_test_a").resolve())
```

Ruff (0.15.4)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux_split_cwd_test_a"

(S108)